swc/private/swc.bzl

```diff
@@ -42,7 +42,14 @@ _attrs = {
         If out_dir is also specified, it is used as the name of the output directory.
         Otherwise, the directory is named the same as the target.""",
     ),
-    "data": js_lib_helpers.JS_LIBRARY_DATA_ATTR,
```

Comment:
Did we delete this?

Reply:
Yes. I opted out of the pattern of exporting individual attributes and their docstrings. Its cleaner and easier to just reference back to the upstream attribute in the downstream attribute docstring.